src/adb/exec.js

```diff
@@ -0,0 +1,15 @@
+let path = require('path');
+let exec = require('mz/child_process').exec;
+let debug = console.log.bind(console, '[adb/exec]');
+let flag = (option, value) => value ? ` ${option} ${value}` : '';
```

Comment:
More nits: What do you think about removing the initial space from this and then putting spaces in the concatenation below?